client/extensions/wp-super-cache/state/settings/actions.js

```diff
@@ -91,6 +92,7 @@ export const saveSettings = ( siteId, settings ) => {
 			{ path: `/jetpack-blogs/${ siteId }/rest-api/` },
 			{ path: '/wp-super-cache/v1/settings', body: JSON.stringify( sanitizeSettings( settings ) ), json: true } )
 			.then( () => {
+				dispatch( requestNotices( siteId ) );
```

Comment:
This is a thunk, i.e. async action, isn't it. Maybe put it after the other two (sync ones) that just update state? I find that'd give me more closure when reading it 😄